doc/source/whatsnew/v1.2.0.rst

```diff
@@ -257,6 +257,7 @@ Other enhancements
 - Improve numerical stability for :meth:`.Rolling.skew`, :meth:`.Rolling.kurt`, :meth:`Expanding.skew` and :meth:`Expanding.kurt` through implementation of Kahan summation (:issue:`6929`)
 - Improved error reporting for subsetting columns of a :class:`.DataFrameGroupBy` with ``axis=1`` (:issue:`37725`)
 - Implement method ``cross`` for :meth:`DataFrame.merge` and :meth:`DataFrame.join` (:issue:`5401`)
+- Improve performance for :meth:`DataFrame.__setitem__` with list-like indexers (:issue:`37954`)
```

Comment:
i think it would be ok to backport this to 1.1.5

Reply:
Moved to 1.1.5